### DISCUSSION TITLE
Quick toggle to HDR mode...

### DISCUSSION THREAD

**Discussion by Noisrevid** [seerge/g-helper#2199] [Ideas]
Hi @seerge any plans to add a button to quick toggle HDR windows mode?    Would be slick to just click a button in Ghelper instead of having to fumble through Windows display settings to enable HDR for when I want to go watch HDR content in Youtube, videos in general and or play a game in HDR mode.     Right now it takes some fumbling through display settings to find it and then back to go back and fumble through settings again to turn it back off since leaving HDR mode on tends to wash out colors overall when you're doing regular things.  

Thanks!

**Comment by seerge** [comment]
@Noisrevid hello, I'm not aware if this is even possible via API or something

Why don't you just use ``Win + Alt + B`` hotkey ?
https://www.pcmag.com/how-to/how-to-play-games-watch-videos-in-hdr-on-windows-10#:~:text=The%20easiest%20way%20to%20enable,down%20and%20toggle%20Use%20HDR.

**Reply by Noisrevid** [reply]
Win+ALT+B does nothing on my 2024 G14...  its possible because i'm using Ghost Spectre Win11 image which is some heavily modified for less resources version of Win11 but it's not working for me.

**Reply by seerge** [reply]
It works perfectly on my G14 2024 :) Just tried that.

What I have noticed HDR toggle is available (also in windows) only when device is plugged by default

**Reply by Noisrevid** [reply]
Just posted a reply that Xbox Game Bar needs to be present.. wonder if there's a way to figure out how to not need it though.

**Reply by seerge** [reply]
P.S. Keep in mind that when HDR is enabled - Flicker Free dimming or Visual Modes won't work (as they don't work in AC).

I don't see much benefits of HDR on OLED screen, as SDR here looks like HDR on others ... + less combability issues in windows

**Reply by seerge** [reply]
With one exception - viewing actual HDR content of course.

**Comment by Noisrevid** [comment]
Turns out Xbox Game Bar needs to be installed for WIN+ALT+B to work.  I guess this will suffice, but it would be nice to be able to figure out how it's not required for Xbox Game Bar to be installed since I never use it and never use Xbox functions in Windows.
